### PR TITLE
Move pkg/subctl/resource to internal/

### PR DESCRIPTION
**Pull request by Jaanki**
This PR moves:
1. pkg/subctl/resource/exec.go -> internal/pods/exec.go
2. pkg/subctl/resource/schedule_pod.go -> internal/pods/schedule.go
3. functions from pkg/subctl/resource/util.go -> internal/nodes/nodes.go

Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: internal/pods/schedule.go [hunk 1]
```diff
@@ -229,3 +229,40 @@ func (np *NetworkPod) AwaitPodCompletion() error {
...
+func addNodeSelectorTerm(nodeSelTerms []v1.NodeSelectorTerm, label string,
+	op v1.NodeSelectorOperator, values []string) []v1.NodeSelectorTerm {
+	return append(nodeSelTerms, v1.NodeSelectorTerm{MatchExpressions: []v1.NodeSelectorRequirement{
+		{
+			Key:      label,
+			Operator: op,
+			Values:   values,
+		},
+	}})
+}
```
Comment (tpantelis) [inline internal/pods/schedule.go]: newline.

File: internal/pods/schedule.go [hunk 2]
```diff
@@ -44,28 +44,28 @@ const (
...
 	Name       string
 	ClientSet  kubernetes.Interface
-	Scheduling PodScheduling
+	Scheduling Scheduling
 	Namespace  string
 	Command    string
 	Timeout    uint
 }
 
 type NetworkPod struct {
```
Comment (tpantelis) [inline internal/pods/schedule.go]: How about:

```suggestion
type Scheduled struct {
```

File: internal/pods/schedule.go [hunk 3]
```diff
@@ -44,28 +44,28 @@ const (
...
 	Pod       *v1.Pod
-	Config    *PodConfig
+	Config    *Config
 	PodOutput string
 }
 
-func SchedulePodAwaitCompletion(config *PodConfig) (string, error) {
+func ScheduleAwaitCompletion(config *Config) (string, error) {
```
Comment (tpantelis) [inline internal/pods/schedule.go]:
```suggestion
func ScheduleAndAwaitCompletion(config *Config) (string, error) {
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: internal/nodes/nodes.go [hunk 4]
```diff
@@ -137,3 +137,4 @@ var nodeLabelBackoff wait.Backoff = wait.Backoff{
 	Factor:   1.2,
 	Jitter:   1,
 }
+
```
Comment (tpantelis) [inline internal/nodes/nodes.go]: Looks like an extraneous change.

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: internal/nodes/nodes.go (hunk 4 above)
Comment (Jaanki) [reply internal/nodes/nodes.go]: yes.

File: pkg/subctl/cmd/gather/cni.go [hunk 5]
```diff
@@ -140,35 +140,35 @@ func gatherOVNResources(info *Info, networkPlugin string) {
 
 	// we check two different labels because OpenShift deploys with a different
 	// label compared to ovn-kubernetes upstream
-	pods, err := findPods(info.ClientSet, ovnMasterPodLabelOCP)
-	if err != nil || pods == nil || len(pods.Items) == 0 {
-		pods, err = findPods(info.ClientSet, ovnMasterPodLabelGeneric)
+	ovnMasterpods, err := findPods(info.ClientSet, ovnMasterPodLabelOCP)
```
Comment (Jaanki) [inline pkg/subctl/cmd/gather/cni.go:143]: This was changed becasue golint complains of shadow variable declaration `pods` of package `pods`

**Review by tpantelis — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**